airbyte-cdk/python/airbyte_cdk/utils/schema_inferrer.py

```diff
@@ -81,7 +92,7 @@ def _clean(self, node: InferredSchema) -> InferredSchema:
                 if len(node["anyOf"]) == 2 and {"type": "null"} in node["anyOf"]:
                     real_type = node["anyOf"][1] if node["anyOf"][0]["type"] == "null" else node["anyOf"][0]
                     node.update(real_type)
-                    node["type"] = [node["type"], "null"]
+                    node["type"] = ["null", node["type"]]
```

Comment:
Nit, non-blocking: would be so nice to have `InferredSchema` provide properties instead of string key access.